core/store/src/columns.rs

```diff
@@ -467,6 +465,8 @@ impl DBCol {
             DBCol::LatestWitnessesByIndex => false,
             // Deprecated.
             DBCol::_ReceiptIdToShardId => false,
+            // This can be re-constructed from the Chunks column, so no need to store in Cold DB.
+            DBCol::PartialChunks => false,
```

Comment:
Does it work magically work?

I think cold DB will throw either an error or a panic if a non cold column is read. Does it mean this column was never read from cold?

Reply:
Yes, it will not be read from cold db, otherwise yes, the binary should panic.